src/main/scala/com/github/codelionx/dodo/actors/ResultCollector.scala

```diff
@@ -77,10 +74,13 @@ class ResultCollector extends Actor with ActorLogging {
     case _ => log.info("Unknown message received")
   }
 
-  def write(message: String): Unit = {
-    bw.write(message + "\n")
-    log.info(message)
-  }
+  def write(message: String): Unit =
+    if (settings.outputToConsole) {
+      bw.write(message + "\n")
+      log.info(message)
+    } else {
+      bw.write(message + "\n")
```

Comment:
This happens whether the if condition is triggered or not and therefore could be done before the check